Makefile

```diff
@@ -194,6 +229,10 @@ endef
 # arch of ppc64le. This canned command will do a last minute transformation of
 # the string we build the "correct" go architecture. However the build result
 # will still be placed at the expected location with names matching ppc64el.
+
+ifeq ($(FORCE_CGO),1)
+run_go_build = $(run_cgo_build)
+else
```

Comment:
I don't get this, why would we want to force CGO built with musl on anything but jujud?